config/routes.rb

```diff
@@ -5,6 +5,8 @@
   resources :users, only: [:index, :edit, :update, :destroy]
   
   resources :trips
+  
+  get 'search_results' => 'trips#search_results', as: 'search_results'
```

Comment:
Move this line above where your resources start, and remove empty lines between resources unless they're nested.